src/Engine/Network/Client.cpp

```diff
@@ -13,6 +13,8 @@ Client::Client(World* world, EventBroker* eventBroker)
     m_PlayerName = config->Get<std::string>("Networking.Name", "Raptorcopter");
     m_SendInputIntervalMs = config->Get<int>("Networking.SendInputIntervalMs", 33);
     LOG_INFO("Client initialized");
+
+    m_ServerlistRequest.Connect(m_PlayerName, "192.168.1.255", 32554);
```

Comment:
This is to establish a temporary connection to get information about all servers and is broadcasted to all servers up and running on the LAN. The port is a random port that we reserve for this purpose, but perhaps we should "config it". The address is the LAN broadcast address but is actually not used, instead we let asio handle the broadcast internally.